packages/mysql/configuration-schema.json

```diff
@@ -1,59 +1,61 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema#",
-    "properties": {
-      "host": {
-        "title": "Host URL",
-        "type": "string",
-        "description": "The database instance host URL",
-        "format": "uri",
-        "minLength": 1,
-        "examples": [
-          "some-host-url.compute-1.amazonaws.com"
-        ]
-      },
-      "port": {
-        "title": "Port",
-        "type": "integer",
-        "default": 3306,
-        "description": "Database instance port",
-        "minLength": 1,
-        "examples": [
-          3306
-        ]
-      },
-      "database": {
-        "title": "Database",
-        "type": "string",
-        "description": "The database name",
-        "minLength": 1,
-        "examples": [
-          "demo-db"
-        ]
-      },
-      "user": {
-        "title": "Username",
-        "type": "string",
-        "description": "The username to log in the database",
-        "minLength": 1,
-        "examples": [
-          "admin-demo"
-        ]
-      },
-      "password": {
-        "title": "Password",
-        "type": "string",
-        "description": "The password to log in the database",
-        "writeOnly": true,
-        "minLength": 1,
-        "examples": [
-          "@super(!)Secretpass"
-        ]
-      }
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "host": {
+      "title": "Host URL",
+      "type": "string",
+      "description": "The database instance host URL",
+      "format": "string",
+      "minLength": 1,
+      "examples": [
+        "some-host-url.compute-1.amazonaws.com"
+      ]
     },
-    "type": "object",
-    "additionalProperties": true,
-    "required": [
-      "host",
-      "database"
-    ]
-  }
\ No newline at end of file
+    "port": {
+      "title": "Port",
+      "type": "integer",
+      "default": 3306,
+      "description": "Database instance port",
+      "minLength": 1,
+      "examples": [
+        3306
+      ]
+    },
+    "database": {
+      "title": "Database",
+      "type": "string",
+      "description": "The database name",
+      "minLength": 1,
+      "examples": [
+        "demo-db"
+      ]
+    },
+    "user": {
+      "title": "Username",
+      "type": "string",
+      "description": "The username to log in the database",
+      "minLength": 1,
+      "examples": [
+        "admin-demo"
+      ]
+    },
+    "password": {
+      "title": "Password",
+      "type": "string",
+      "description": "The password to log in the database",
+      "writeOnly": true,
+      "minLength": 1,
+      "examples": [
+        "@super(!)Secretpass"
+      ]
+    }
+  },
+  "type": "object",
+  "additionalProperties": true,
+  "required": [
+    "host",
+    "database",
+    "user",
```

Comment:
Are we sure user and pass are required? Is this a behaviour change?

Reply:
Yes, it's a database connection, You must have a username and password